.evergreen/patch/diff.patch

```diff
@@ -0,0 +1,1327 @@
+diff --git a/test/load_balancer/cursors.json b/test/load_balancer/cursors.json
+index 27aaddd5..b11bf2c6 100644
+--- a/test/load_balancer/cursors.json
++++ b/test/load_balancer/cursors.json
```

Comment:
In standup we decided this pattern would be more maintainable if we split each change into its own patch file referencing the PYTHON ticket which would eventually remove it. Like PYTHON-XXX.patch.